### PR TITLE
Update on Chapter01.MD based first PR comments

### PR DESCRIPTION
**Pull request by jeffsaelens**
Adding my thoughts into a PR based on feedback from others.

### CODE REVIEW

File: whitepaper/chapter01.md [hunk 1]
```diff
@@ -44,13 +44,15 @@ There are some additional key principles to cloud native that are important to g
...
+Cloud native systems have a clear separation between the microservices. The separation is based on a technology like containers which allows independent life cycle management of each microservice. Cloud native applications, which are built on these microservices, and systems generally benefit from design considerations that segregate services where appropriate by leveraging technology such as containers. Additional benefits of cloud native architectures include scalability (vertical and horizontal) per microservice. This simplifies deployment in a wide variety of infrastructure environments, ranging from endpoint device to edge station to large data centres. It also allows service providers to deploy resources in the most efficient manner possible while maintaining necessary service levels for regulated and unregulated service types.
```
Comment (taylor) [inline whitepaper/chapter01.md]: > design considerations that segregate services where appropriate by leveraging technology such as containers.

It seems like _where appropriate_ is too vague. Can we leverage something from the [cloud native microservices principles](https://networking.cloud-native-principles.org/cloud-native-microservice-principles) to expand or reword this?

File: whitepaper/chapter01.md [hunk 2]
```diff
@@ -44,13 +44,15 @@ There are some additional key principles to cloud native that are important to g
...
+While one microservice per container is the ideal state for cloud native applications, the reality is not always so clean and simple. It is entirely possible and sometimes required to run multiple microservices in a single container, and even to deploy very large containers for aggregations of services that are difficult or impossible to decouple due to software or network limitations. However, the general idea behind cloud native is to decompose microservices into the smallest possible infrastructure unit allowable. This causes a shift in where the burden of orchestration lies, and pushes it solely (ideally) into the orchestration layer.
```
Comment (taylor) [inline whitepaper/chapter01.md]: I like the expansion on the second half with the orchestration layer.

> However, the general idea behind cloud native is to decompose microservices into the smallest possible infrastructure unit allowable.
For the decomposing sentence what drives "what is _allowable_?"   It's lost the reason when splitting.  Something needs to be added to that sentence, IMO.

File: whitepaper/chapter01.md [hunk 3]
```diff
@@ -76,7 +80,7 @@ Cloud native applications consist of sets of cloud native microservices. The con
 
 A cloud native network function (CNF) is a cloud native application that implements or facilitates network functionality. A cloud native network function consists of one or more microservices, and has been developed using Cloud Native Principles including immutable infrastructure, declarative APIs and a “repeatable deployment process”.
 
-An example of a simple CNF is a packet filter which implements a single piece of network functionality as a microservice. A firewall is an example of a CNF which may be composed of more than one microservice.
+An example of a simple CNF is a packet filter which implements a single piece of network functionality as a microservice. A firewall is an example of a CNF which may be composed of more than one microservice (i.e. encryption, decryption, access lists, packet inspection etc...).
```
Comment (taylor) [inline whitepaper/chapter01.md]: :+1:

File: whitepaper/chapter01.md [hunk 4]
```diff
@@ -44,13 +44,15 @@ There are some additional key principles to cloud native that are important to g
...
+Cloud native applications also have all of their runtime dependencies packaged with them during the build phase which is then used for deployment. This allows for individual business units and operations teams to care and maintain for their software with less risk of other stakeholders disrupting their service’s “uptime”. In practice, this allows for scenarios such as owner of service “X” being able to theoretically scale their chosen software independent of service “Y”. 
+
+These potentially clear lines of demarcation provide a paradigm that makes sense to legacy operations groups who are being forced to rapidly assess how they’ve delegated roles and responsibilities. For example, a network focused operations group could do upgrades and updates to a microservice hosting routing protocol services such as ISIS or BGP, without impacting the development or operations of the security team managing the microservices hosting AAA or Access List functionality.
```
Comment (taylor) [inline whitepaper/chapter01.md]: :+1:

File: whitepaper/chapter01.md [hunk 5]
```diff
@@ -59,6 +61,8 @@ Because they are loosely coupled, cloud native services are always managed by an
...
 
+Kubernetes is unique in that it provides an example of both a declarative orchestration model, as well as an application that can be deployed and consumed in a cloud native manner. From an orchestration standpoint, Kubernetes presents a declarative API and works off an assumption of immutability. Conversely, Kubernetes is an application that is comprised of a decoupled system of microservices that are deployed via orchestration. Given these design implementations, Kubernetes itself is often deployed via Kubernetes and treated the same as any other cloud native application. The expectations, set forth by this example, would be that a CNF would behave in a similar manner, with its core componentry being immutable and its method of configuration being declarative.
```
Comment (taylor) [inline whitepaper/chapter01.md]: :+1:

File: whitepaper/chapter01.md [hunk 6]
```diff
@@ -97,7 +101,9 @@ This fostered a “top-down” approach to telecommunications systems that were
...
...This next evolution will require the decoupling of tightly bundled functions and processes to create a more resilient, manageable, and performant infrastructure that derives immutability precisely from its ephemerality. For these considerations to have any hope of adoption, CNFs will have to first be introduced alongside VNFs and PNFs (physical network functions) and will need to fit into operational models that govern those earlier types of network functions. Finding sensible integration points between emerging green fields and legacy brown fields will be crucial to the success and adoption of cloud native practices at larger telcos. CTO offices and architecture groups must find ways to stitch CNFs into existing layer 2 and 3 domains, integrate CNFs into existing management systems currently present in the service providers' networks as well as find ways to introduce some of the continuous integration and deployment/delivery practices to the PNF/VNF world. These achievable objectives are crucial as they provide executives both a means of capturing a return on investment with their existing infrastructure and a realistic path forward in taking a more holistic cloud native approach.
```
Comment (taylor) [inline whitepaper/chapter01.md]: :+1:

File: whitepaper/chapter01.md (hunk 2 above)
Comment (jeffsaelens) [reply whitepaper/chapter01.md]: I cleaned up the earlier comment on appropriate segregation, but I need some recommendations on how to provide context to this specific concept. This is a holdover from the original draft, and I'm not 100% sure how to scope the concept of an "allowable unit".

Comment (tomkivlin) [reply whitepaper/chapter01.md]: Linking to "even to deploy very large containers for aggregations of services that are difficult or impossible to decouple due to software or network limitations" - is the "allowable unit" linked to those "software or network limitations"? And if so, what do we think those limitations are?

File: whitepaper/chapter01.md [hunk 7]
```diff
@@ -97,7 +101,9 @@ This fostered a “top-down” approach to telecommunications systems that were
...
+This was the same pattern followed during server virtualisation, when initial efforts focused on replicating physical servers as virtual entities. Cloud native however, requires a more fundamental redesign and rethink of how telco and backbone/core network systems function. This shift is not limited to only the software architectures being proposed but also how the telco operator structures and organizes its staff.
```
Comment (tomkivlin) [inline whitepaper/chapter01.md]: 👍 👍

**Pull request merged**